src/components/image/index.test.tsx

```diff
@@ -309,6 +336,30 @@ describe("Image component", () => {
         );
       });
     });
+    it("Should call saveSolidDatasetAt when clicking delete button", async () => {
+      jest
+        .spyOn(SolidFns, "saveSolidDatasetAt")
+        .mockResolvedValue(mockDataset as any);
```

Comment:
Adding 
```
import type {
  SolidDataset,
  WithChangeLog,
  WithServerResourceInfo,
} from "@inrupt/solid-client";
```
in the imports and replacing this with `mockDataset as SolidDataset & WithServerResourceInfo & WithChangeLog` should remove the warning.